cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -298,6 +299,59 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    SubsectionShareLinkXBlockModal = CourseOutlineXBlockModal.extend({
+
+        initializeEditors: function() {
+            var tabsTemplate;
+            var tabs = this.options.tabs;
+            if (tabs && tabs.length > 0) {
+                    tabsTemplate = this.loadTemplate('subsection-share-link-modal-tabs');
+                    HtmlUtils.setHtml(this.$('.modal-section'), HtmlUtils.HTML(tabsTemplate({tabs: tabs})));
+                    _.each(this.options.tabs, function(tab) {
+                        // eslint-disable-next-line prefer-spread
+                        this.options.editors.push.apply(
+                            this.options.editors,
+                            _.map(tab.editors, function(Editor) {
+                                return new Editor({
+                                    parent: this,
+                                    parentElement: this.$(`.modal-section .${tab.name}`),
+                                    model: this.model,
+                                    xblockType: this.options.xblockType,
+                                });
+                            }, this)
+                        );
+                    }, this);
+                    this.showTab(tabs[0].name);
+            } else {
+                CourseOutlineXBlockModal.prototype.initializeEditors.call(this);
+            }
+        },
+
+        events: _.extend({}, CourseOutlineXBlockModal.prototype.events, {
+            'click .subsection-share-link-tab-button': 'handleShowTab'
+        }),
+
+        getTitle: function() {
+            return gettext('Share Mechanics and Biomechanics subsection');
+        },
+
+        addActionButtons: function() {
+            this.addActionButton('cancel', gettext('Close'));
+        },
+
+        handleShowTab: function(event) {
+            event.preventDefault();
+            this.showTab($(event.target).data('tab'));
+        },
+
+        showTab: function(tab) {
+            this.$('.modal-section .subsection-share-link-tab-button').removeClass('active');
+            this.$('.modal-section .subsection-share-link-tab-button[data-tab="' + tab + '"]').addClass('active');
+            this.$('.modal-section .subsection-share-link-tab').hide();
+            this.$('.modal-section .' + tab).show();
```

Comment:
Use template strings for these modal-section
```suggestion
          this.$(`.modal-section .subsection-share-link-tab-button[data-tab="${tab}"]`).addClass('active');
          this.$(`.modal-section .subsection-share-link-tab`).hide();
          this.$(`.modal-section .${tab}`).show();
```

Reply:
Done